pele_platform/Frag/libraries.py

```diff
@@ -1,36 +1,54 @@
 import glob
 import os
 import subprocess
+import shutil
+import tempfile
 
+from rdkit import Chem
 from pele_platform.constants import constants as cs
 import pele_platform.Utilities.Helpers.helpers as hp
 
 OUTPUT = "input.conf"
 
+
+def getSymmetryGroups(mol):
+    """
+    Computes the symmetry class for each atom and returns a list with the idx of non-symmetric atoms.
+    """
+    rank = {} 
+    symmetryList=[] 
+    symmetryRankList=[]
+    counter=0
+    
+    for atom in mol.GetAtoms():
+        rank[atom.GetIdx()] = list(Chem.CanonicalRankAtoms(mol,breakTies=False))[counter]
+        counter += 1
```

Comment:
In Python there is the `enumerate()` Python that allows you to forget about using counters like the one you set. It is as easy as:
```python
for counter, atom in enumerate(mol.GetAtoms()):
    rank[atom.GetIdx()] = list(Chem.CanonicalRankAtoms(mol,breakTies=False))[counter]
    # And counter will be equal to the iteration number
    # If you are only interested in the number of iterations you did, it is even easier:
    # counter = len(mol.GetAtoms()) 
```